pkg/asset/installconfig/gcp/validation.go

```diff
@@ -181,20 +208,25 @@ func validateInstanceTypes(client API, ic *types.InstallConfig) field.ErrorList
 	zones := defaultZones
 	instanceType := defaultInstanceType
 	arch := types.ArchitectureAMD64
-	cpDiskType := ""
+	cpDiskType := defaultDiskType
 	if ic.ControlPlane != nil {
 		arch = string(ic.ControlPlane.Architecture)
 		if instanceType == "" {
 			instanceType = DefaultInstanceTypeForArch(ic.ControlPlane.Architecture)
 		}
+		if cpDiskType == "" {
```

Comment:
Seems that this check may be unnecessary since defaultDiskType, and therefore cpDiskType, will never be set to ""